environment-api/src/test/java/com/sequenceiq/environment/api/v1/environment/model/request/aws/AwsDiskEncryptionParametersTest.java

```diff
@@ -0,0 +1,84 @@
+package com.sequenceiq.environment.api.v1.environment.model.request.aws;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import static org.junit.Assert.assertFalse;
```

Comment:
please do not mix Junit4 and Junit5, use Junit5's Assertions.assertFalse and assertTrue